internal/provider/waypoint/data_source_waypoint_application.go

```diff
@@ -100,6 +101,31 @@ func (d *DataSourceApplication) Schema(ctx context.Context, req datasource.Schem
 					},
 				},
 			},
+			"output_values": schema.ListNestedAttribute{
+				Computed: true,
+				Description: "The output values of the Terraform run for the Add-on, Sensitive values have type " +
+					"and value omitted.",
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"name": schema.StringAttribute{
+							Computed:    true,
+							Description: "The name of the output value.",
+						},
+						"type": schema.StringAttribute{
```

Comment:
Does terraform not have a more strict schema here for output variable values? Basically someone could type anything here right now and it would accept it (or make a typo).

Reply:
I am a little confused by your question, because the output values are always computed the user shouldn't be typing anything here.

Reply:
Ah, I see it's computed, yes nevermind 👍🏻